src/Sparrow/Json/AbstractBlittableJsonTextWriter.cs

```diff
@@ -19,60 +20,74 @@ public abstract unsafe class AbstractBlittableJsonTextWriter : IBlittableJsonTex
         private const byte Comma = (byte)',';
         private const byte Quote = (byte)'"';
         private const byte Colon = (byte)':';
-        public static readonly byte[] NaNBuffer = { (byte)'"', (byte)'N', (byte)'a', (byte)'N', (byte)'"' };
 
-        public static readonly byte[] PositiveInfinityBuffer =
-        {
-           (byte)'"', (byte)'I', (byte)'n', (byte)'f', (byte)'i', (byte)'n', (byte)'i', (byte)'t', (byte)'y', (byte)'"'
+        public static ReadOnlySpan<byte> NewLineBuffer => "\r\n"u8;
+        public static ReadOnlySpan<byte> NaNBuffer => "\"NaN\""u8;
+        public static ReadOnlySpan<byte> PositiveInfinityBuffer => "\"Infinity\""u8;
+        public static ReadOnlySpan<byte> NegativeInfinityBuffer => "\"-Infinity\""u8;
+
+        public static readonly byte[] NullBuffer = "null"u8.ToArray();
+        public static readonly byte[] TrueBuffer = "true"u8.ToArray();
+        public static readonly byte[] FalseBuffer = "false"u8.ToArray();
+
+        /// <summary>
+        /// The original code that generates this flatten sequence.
+        /// ControlCodeEscapes = new byte[32][];
+        /// for (int i = 0; i< 32; i++)
+        /// {
+        ///    ControlCodeEscapes[i] = Encodings.Utf8.GetBytes(i.ToString("X4"));
+        /// }
+        /// 
+        /// </summary>
+        private static ReadOnlySpan<byte> _controlCodeEscapes => "0000000100020003000400050006000700080009000A000B000C000D000E000F0010001100120013001400150016001700180019001A001B001C001D001E001F"u8;
+
+        internal static ReadOnlySpan<int> ControlCodeEscapes => MemoryMarshal.Cast<byte, int>(_controlCodeEscapes);
+
+        /// <summary>
+        /// The original code that generates this flatten sequence.
+        /// EscapeCharacters = new byte[256];
+        /// for (int i = 0; i< 32; i++)
+        ///     EscapeCharacters[i] = 0;
+        ///
+        /// for (int i = 32; i<EscapeCharacters.Length; i++)
+        ///     EscapeCharacters[i] = 255;
+        ///
+        /// EscapeCharacters[(byte)'\b'] = (byte)'b';
+        /// EscapeCharacters[(byte)'\t'] = (byte)'t';
+        /// EscapeCharacters[(byte)'\n'] = (byte)'n';
+        /// EscapeCharacters[(byte)'\f'] = (byte)'f';
+        /// EscapeCharacters[(byte)'\r'] = (byte)'r';
+        /// EscapeCharacters[(byte)'\\'] = (byte)'\\';
+        /// EscapeCharacters[(byte)'/'] = (byte)'/';
+        /// EscapeCharacters[(byte)'"'] = (byte)'"';
+        /// </summary>
+        private static ReadOnlySpan<byte> EscapeCharacters => new[]
```

Comment:
```suggestion
        private static ReadOnlySpan<byte> EscapeCharacters => new byte[]
```

And then you save all the `(byte)` casts

Reply:
LOL that was a pretty dumb thing... As I just wrote the code to write that table, didn't see it coming.